gsdk/src/metadata/generated.rs

```diff
@@ -257,109 +259,84 @@ pub mod runtime_types {
             pub mod pallet {
                 use super::runtime_types;
                 #[derive(Debug, crate::gp::Decode, crate::gp::DecodeAsType, crate::gp::Encode)]
-                #[doc = "Contains a variant per dispatchable extrinsic that this pallet has."]
                 pub enum Call {
                     #[codec(index = 0)]
-                    #[doc = "See [`Pallet::remark`]."]
                     remark {
-                        remark: ::std::vec::Vec<::core::primitive::u8>,
+                        remark: ::subxt::ext::subxt_core::alloc::vec::Vec<::core::primitive::u8>,
                     },
                     #[codec(index = 1)]
-                    #[doc = "See [`Pallet::set_heap_pages`]."]
                     set_heap_pages { pages: ::core::primitive::u64 },
                     #[codec(index = 2)]
-                    #[doc = "See [`Pallet::set_code`]."]
                     set_code {
-                        code: ::std::vec::Vec<::core::primitive::u8>,
+                        code: ::subxt::ext::subxt_core::alloc::vec::Vec<::core::primitive::u8>,
                     },
                     #[codec(index = 3)]
-                    #[doc = "See [`Pallet::set_code_without_checks`]."]
                     set_code_without_checks {
-                        code: ::std::vec::Vec<::core::primitive::u8>,
+                        code: ::subxt::ext::subxt_core::alloc::vec::Vec<::core::primitive::u8>,
                     },
                     #[codec(index = 4)]
-                    #[doc = "See [`Pallet::set_storage`]."]
                     set_storage {
-                        items: ::std::vec::Vec<(
-                            ::std::vec::Vec<::core::primitive::u8>,
-                            ::std::vec::Vec<::core::primitive::u8>,
+                        items: ::subxt::ext::subxt_core::alloc::vec::Vec<(
+                            ::subxt::ext::subxt_core::alloc::vec::Vec<::core::primitive::u8>,
+                            ::subxt::ext::subxt_core::alloc::vec::Vec<::core::primitive::u8>,
                         )>,
                     },
                     #[codec(index = 5)]
-                    #[doc = "See [`Pallet::kill_storage`]."]
                     kill_storage {
-                        keys: ::std::vec::Vec<::std::vec::Vec<::core::primitive::u8>>,
+                        keys: ::subxt::ext::subxt_core::alloc::vec::Vec<
+                            ::subxt::ext::subxt_core::alloc::vec::Vec<::core::primitive::u8>,
+                        >,
                     },
                     #[codec(index = 6)]
-                    #[doc = "See [`Pallet::kill_prefix`]."]
                     kill_prefix {
-                        prefix: ::std::vec::Vec<::core::primitive::u8>,
+                        prefix: ::subxt::ext::subxt_core::alloc::vec::Vec<::core::primitive::u8>,
```

Comment:
why can't we use std types?

Reply:
or just alloc

Reply:
code in `runtime_types` are generated by `subxt-codegen`, imo we'd better not hack it too much or it will be hard to maintain in the future xd